scheduler/docs/rebalancer-config.asc

```diff
@@ -0,0 +1,52 @@
+== Rebalancer Configuration
+
+Cook's preemption is performed by the Rebalancer.  The Rebalancer is distinct from the Scheduler, which of course is responsible for actually scheduling jobs.
+
+The Rebalancer periodically examines all running Cook tasks on the cluster, together with all of the pending jobs.  Combining these two sets of jobs, Cook imagines alternate permutations of tasks that could be running on the cluster.  If it finds a permutation that represents a significant improvement (in terms of resource usage balance between users) over the set of tasks that are currently running, the Rebalancer will kill currently running tasks that are not a part of the desirable set of running tasks.
+
+Importantly, the Rebalancer does not actually schedule the pending jobs that are a part of the desirable permuation; the idea is that the Scheduler can be trusted to do a better job to improve balance, given a second chance with the new room that is Rebalancer is creating by pre-empting jobs.
+
+For more detailed information about the design of the Rebalancer, see the source code link:../src/cook/mesos/rebalancer.clj[here]:
+
+=== How the Configuration is stored
+
+Currently, Rebalancer configuration is stored only in Datomic.  In Cook's database,
+Rebalancer configuration options are facts about the entity with id :rebalancer/config.  Thus, to configure the Rebalancer, you will need to run code like this:
+
+[source, clojure]
+----
+(let [conn (d/connect "(your-cook-database-url)")
+        db (d/db conn)]
+    @(d/transact conn [{:db/id :rebalancer/config
+                        :rebalancer.config/min-utilization-threshold 0.0
+                        :rebalancer.config/safe-dru-threshold 0.0
+                        :rebalancer.config/min-dru-diff 0.0000000001
+                        :rebalancer.config/max-preemption 64.0}]))
```

Comment:
Actually, I never set any shares whatsoever!
So, my divisors were universally Double.MAX_VALUE.

I can add documentation about this.